easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -87,6 +87,48 @@ def setup_cmake_env(tc):
     setvar("CMAKE_LIBRARY_PATH", library_paths)
 
 
+def setup_cmake_env_python_hints(cmake_version):
```

Comment:
The argument looks unused? If you want to use it then I'd suggest this and below:
```suggestion
def setup_cmake_env_python_hints(cmake_version=None):
```

Reply:
I had forgotten to remove it but i think it might not hurt to allow it to be passed [7aff4c0](https://github.com/easybuilders/easybuild-easyblocks/pull/3463/commits/7aff4c0026a6c5e4b2b0a605cef4e847b698886b)